packages/@aws-cdk/aws-amplify/README.md

```diff
@@ -208,3 +208,19 @@ const amplifyApp = new amplify.App(stack, 'App', {
   ],
 });
 ```
+
+## Deploying Assets
+
+`sourceCodeProvider` is optional; when this is not specified the Amplify app can be deployed to using `.zip` packages. The `AmplifyAssetDeployment` construct can be used to deploy S3 assets to Amplify as part of the CDK:
+
+```ts
+const asset = new assets.Asset(this, "SampleAsset", {});
+const amplifyApp = new amplify.App(this, 'MyApp', {});
+const branch = amplifyApp.addBranch("dev");
+new AmplifyAssetDeployment(this, "AmplifyAssetDeployment", {
+  app: amplifyApp,
+  branch: branch,
+  s3BucketName: asset.s3BucketName,
+  s3ObjectKey: asset.s3ObjectKey,
+});
```

Comment:
what about this kind of API?

```suggestion
branch.deployAsset(asset);
```

you could also have 
```ts
branch.deployUrl('https://www.publicly-accessible/file.zip');
```

or `branch.deploy(Artifacts.fromS3Asset(...))` / `branch.deploy(Artifacts.fromUrl(...))`

Reply:
Great suggestion @jogold ; I can make this change. I will just do it for Assets for now and can allow URLs in a future PR if that's fine with you?

Reply:
> if that's fine with you?

Of course! I'm not sure about the naming though... maybe `branch.addAssetDeployment()`

Anyway this can be easily changed later after @MrArnoldPalmer reviews this.

Reply:
Change made :)